packages/plugins/prometheus/package.json

```diff
@@ -42,7 +42,7 @@
     "prom-client": "^15.0.0"
   },
   "dependencies": {
-    "@envelop/prometheus": "11.0.0"
+    "@envelop/prometheus": "12.0.0-alpha-20241112124749-b05f813adacaed7f3a2132da52552b4511ccbc15"
```

Comment:
Yes, but I want to wait for both Yoga and Mesh versions to be ready before actually releasing, in case I have something to change again during the implementation of those